src/components/ImportOtterDialog.tsx

```diff
@@ -14,19 +14,18 @@ import {
   createBlock,
   deleteBlock,
   getBasicTreeByParentUid,
-  getChildrenLengthByPageUid,
   getOrderByBlockUid,
   getParentUidByBlockUid,
-  getTreeByPageName,
-} from "roam-client";
+  toRoamDate,
+} from "roam-client"
 import {
   createOverlayRender,
   getSettingValueFromTree,
   getSubTree,
   setInputSetting,
-  useSubTree,
 } from "roamjs-components";
 import format from "date-fns/format";
+import {addDays} from "../date"
```

Comment:
date-fns have a AddDays function: https://date-fns.org/v2.28.0/docs/addDays

Reply:
ah, good to know! I always have to implement it everywhere 🙈 Kind of dumb that js doesn't have this handled in standard library